src/main/java/io/github/nucleuspowered/nucleus/commands/kit/KitCommand.java

```diff
@@ -0,0 +1,102 @@
+/*
+ * This file is part of Nucleus, licensed under the MIT License (MIT). See the LICENSE.txt file
+ * at the root of this project for more details.
+ */
+package io.github.nucleuspowered.nucleus.commands.kit;
+
+import com.google.common.collect.Lists;
+import com.google.inject.Inject;
+import io.github.nucleuspowered.nucleus.Util;
+import io.github.nucleuspowered.nucleus.api.PluginModule;
+import io.github.nucleuspowered.nucleus.config.KitsConfig;
+import io.github.nucleuspowered.nucleus.internal.CommandBase;
+import io.github.nucleuspowered.nucleus.internal.Kit;
+import io.github.nucleuspowered.nucleus.internal.annotations.Modules;
+import io.github.nucleuspowered.nucleus.internal.annotations.Permissions;
+import io.github.nucleuspowered.nucleus.internal.annotations.RegisterCommand;
+import io.github.nucleuspowered.nucleus.internal.interfaces.InternalNucleusUser;
+import io.github.nucleuspowered.nucleus.internal.permissions.SuggestedLevel;
+import io.github.nucleuspowered.nucleus.internal.services.datastore.UserConfigLoader;
+import org.spongepowered.api.command.CommandResult;
+import org.spongepowered.api.command.args.CommandContext;
+import org.spongepowered.api.command.args.GenericArguments;
+import org.spongepowered.api.command.spec.CommandSpec;
+import org.spongepowered.api.entity.living.player.Player;
+import org.spongepowered.api.item.inventory.Inventory;
+import org.spongepowered.api.text.Text;
+
+import java.util.List;
+
+/**
+ * Allows a user to warp to the specified warp.
+ *
+ * Command Usage: /kit Permission: nucleus.kit.base
+ */
+@Permissions(suggestedLevel = SuggestedLevel.ADMIN)
+@Modules(PluginModule.KITS)
+@RegisterCommand("kit")
+public class KitCommand extends CommandBase<Player> {
+
+    private final String kit = "kit";
+
+    @Inject private KitsConfig kitConfig;
+    @Inject private UserConfigLoader userConfigLoader;
+
+    @Override
+    public CommandSpec createSpec() {
+        return CommandSpec.builder().executor(this).children(this.createChildCommands())
+                .arguments(GenericArguments.onlyOne(GenericArguments.string(Text.of(kit)))).build();
```

Comment:
@dualspiral I'm not familiar with making custom argument parsers, how would I do so?